src/wp-includes/SimplePie/Sanitize.php

```diff
@@ -302,7 +302,7 @@ protected function is_https_domain($domain)
 	}
 
 	/**
-	 * Force HTTPS for selected Web sites.
+	 * Force HTTPS for selected websites.
```

Comment:
Because [SimplePie](https://github.com/simplepie/simplepie) is maintained separately (e.g. [here is the Trac ticket to upgrade Core to use SimplePie 1.7.0](https://core.trac.wordpress.org/ticket/55604)), I believe this should be omitted and addressed upstream.

Reply:
Agreed - thanks @ironprogrammer

Reply:
Once removed, I can approve this PR 👍🏻